board/docker-compose.yml

```
@@ -1,6 +1,6 @@
 services:
   board:
-    image: k8s-study/board:1.1
+    image: k8s-study/board:1.0
```

해당 파일은 이번 과제의 수정 대상이 아닙니다. 롤백해 주세요!
```suggestion
    image: k8s-study/board:1.1
```